docs/content/en/docs/reference/clusterspec/cni.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "cni plugin configuration"
+linkTitle: "cni"
+weight: 20
+description: >
+EKS Anywhere cluster yaml cni plugin specification reference
+---
+
+### Specifying CNI Plugin in EKS Anywhere cluster spec
+
+EKS Anywhere currently supports two CNI plugins: Cilium and Kindnet. Only one of them can be selected
+for a cluster, and the plugin cannot be changed once the cluster is created.
+Up until the 0.7.x releases, the plugin had to be specified using the `cni` field on cluster spec.
+Starting release 0.8, the plugin should be specified using the new `cniConfig` field as follows:
+
+- For selecting Cilium as the CNI plugin:
```

Comment:
After the bullet, I suggest indenting each line of the yaml file and other text three spaces. That will allow it to be indented under the bullet.